MAUI/Scheduler/reminders.md

```diff
@@ -0,0 +1,244 @@
+/// </summary>
+public class Reminder
+{
+    /// <summary>
+    /// Gets or sets a value that decides to notify the reminder before the appointment’s start time.
+    /// </summary>
+    public TimeSpan TimeBeforeStart { get; set; }
+    
+    /// <summary>
+    /// Gets or sets the value indicating whether the reminder is dismissed or not. 
+    /// </summary>
+    public bool Dismissed { get; set; }
+
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+Map those properties of the `Event` class with the [SfScheduler] control by using the [AppointmentMapping] and map `CustomReminder` properties with the [SchedulerReminder] by using [ReminderMapping.]
+
+{% tabs %}
+{% highlight xaml tabtitle="MainWindow.xaml" hl_lines="3 17 18 19 20" %}
+ <syncfusion:SfScheduler x:Name="Schedule" 
+                ItemsSource="{Binding Events}"
+                EnableReminder="True">
+            <syncfusion:SfScheduler.AppointmentMapping>
+                <syncfusion:AppointmentMapping
+                    Subject="EventName"
+                    StartTime="From"
+                    EndTime="To"
+                    AppointmentBackground="Color"
+                    IsAllDay="IsAllDay"
+                    StartTimeZone="StartTimeZone"
+                    EndTimeZone="EndTimeZone"
+                    RecurrenceExceptionDates="RecurrenceExceptions"
+                    RecurrenceRule="RecurrenceRule"
+                    RecurrenceId="RecurrenceId"
+                    Reminders="Reminders">
+                    <syncfusion:AppointmentMapping.ReminderMapping>
+                        <syncfusion:ReminderMapping IsDismissed="Dismissed"
+                                                    TimeBeforeStart="TimeBeforeStart"/>
+                    </syncfusion:AppointmentMapping.ReminderMapping>
+                </syncfusion:AppointmentMapping>
+            </syncfusion:SfScheduler.AppointmentMapping>
+        </syncfusion:SfScheduler>
+{% endhighlight %}
+{% highlight c# tabtitle="ReminderViewModel.cs" %}
+public class ReminderViewModel 
+{
+  ...
+  public ObservableCollection<Event> Events { get; set; } = new ObservableCollection<Event>();
+  this.Events.Add(new Event()
+  {
+    From = DateTime.Now,
+    To = DateTime.Now.AddHours(1),
+    Color = new SolidColorBrush((Color)ColorConverter.ConvertFromString("#FF339933")),
+    EventName = "Conference",
+    Reminders = new ObservableCollection<Reminder>
+    {
+        new Reminder { TimeBeforeStart = new TimeSpan(0)},
+    }
+    });
+}
+{% endhighlight %}
+{% endtabs %}
+
+## ReminderAlertOpening event
+Scheduler notify the appointment's reminder by [ReminderAlertOpening] event before the appointment's start time. The [ReminderAlertOpeningEventArgs] has the following property,
+* [Reminders]: Gets a list of reminders that are used to notify the appointment reminders.
+
+{% tabs %}
+{% highlight c# tabtitle="MainWindow.xaml.cs" hl_lines="1" %}
+scheduler.ReminderAlertOpening += Scheduler_ReminderAlertOpening;
+
```

Comment:
Handle Dismissed for reminder
•	Normal appointment directly dismissed using 

•	For recurrence appointment, if current occurrence need to dismiss then need to add changed occurrence for reminder occurrence dismissed, then current occurrence dismissed details get updated in underlying appointment or data source. 
•	If only occurrence dismissed, then the changed icon will not be updated for dismissed changed occurrence 
Handle Snooze for reminder

Reply:
Updated the content.